### PR TITLE
Move NTP settings in values.yaml into `.network`

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/roadmap/issues/2132

This PR adapts `values.yaml` and the template to match the values schema, where the NTP settings are placed within the `.network` object.

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] Update `/examples` if required.

### CODE REVIEW

**Review by erkanerol — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix `values.yaml` to set NTP settings in `.network.ntp` and adapt template.
```
Comment (kopiczko) [inline CHANGELOG.md:12]: This isn't a fix. This is a breaking change and that should be boldly noted in the changelog.

Comment (marians) [reply CHANGELOG.md:12]: ACK, I'll provide a PR to improve the changelog.

Comment (marians) [reply CHANGELOG.md:12]: FYI I saw it as a fix because the change is an alignment between values.yaml and values.schema.json, which were out of sync.

Comment (marians) [reply CHANGELOG.md:12]: Here: https://github.com/giantswarm/cluster-cloud-director/pull/134

**Pull request merged**